code/datums/elements/strippable.dm

```diff
@@ -122,7 +122,7 @@
 	if (isnull(item))
 		return FALSE
 
-	if (user.action_busy && !skillcheck(user, SKILL_POLICE, SKILL_POLICE_SKILLED))
+	if ((user.action_busy && !skillcheck(user, SKILL_POLICE, SKILL_POLICE_SKILLED)) || user.is_mob_restrained() || HAS_TRAIT(user, (TRAIT_IMMOBILIZED || TRAIT_INCAPACITATED || TRAIT_FLOORED)))
```

Comment:
You can't use HAS_TRAIT this way. Even if it was a bitfield (its not - its a index lookup in a list) you'd use a bitwise or not a boolean or.

```suggestion
	if (user.action_busy && !skillcheck(user, SKILL_POLICE, SKILL_POLICE_SKILLED))
		to_chat(user, SPAN_WARNING("You can't do this right now."))
		return FALSE

	if (user.is_mob_incapacitated())
		to_chat(user, SPAN_WARNING("You can't do this right now."))
		return FALSE

	if (HAS_TRAIT(user, TRAIT_IMMOBILIZED) || HAS_TRAIT(user, TRAIT_FLOORED))
		to_chat(user, SPAN_WARNING("You can't do this right now."))
		return FALSE
```

Reply:
ah thanks